A32NX/html_ui/Pages/A32NX_Core/A32NX_TransitionAltitude.js

```diff
@@ -0,0 +1,348 @@
+            this.checkStart = false;
+        } else if ((this.checkStartSec >= 60 * 1000) && (this.testAirport !== "")) {
+            this.offline = false;
+            this.checkStart = false;
+            this.checkStartSec = 0;
+        }
+    }
+
+    offlineTransAlt() {
+        const Departure = NXDataStore.get("PLAN_ORIGIN", "");
+        const Arrival = NXDataStore.get("PLAN_DESTINATION", "");
+        if (Departure !== "" && Arrival !== "") {
+            this.departureLogic(Departure);
+            this.arrivalLogic(Arrival);
+            if (this.currentDeparture !== Departure) {
+                this.departureLogic(Departure);
+                this.currentDeparture = Departure;
+            }
+            if (this.currentArrival !== Arrival) {
+                this.arrivalLogic(Arrival);
+                this.currentArrival = Arrival;
+            }
+        }
+    }
+
+    departureLogic(airport) {
+        const airportDepart_1 = airport.substr(0,1);
+        const airportDepart_2 = airport.substr(0,2);
+        if (airportDepart_1 === "K" || airportDepart_1 === "C") {
+            SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", 18000); //Canada & USA fixed to 18,000ft
+        } else if (airportDepart_1 === "Y") {
+            SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", 10000); //SPA region only have 10,000ft & 11,000ft. But 10,000ft is more so using that
+        } else {
+            let departure = airportData.find(airportData => airportData.icao === airportDepart_2);
+            if (departure.icao !== airportDepart_2) {
+                SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", 18000); // if can't find on list, default value to 18,000ft
+            } else {
+                SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", departure.transAlt);
+            }
+        }
+    }
+    arrivalLogic(airport) {
+        const airportArrive_1 = airport.substr(0,1);
+        const airportArrive_2 = airport.substr(0,2);
+        if (airportArrive_1 === "K" || airportArrive_1 === "C") {
+            SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", 18000); //Canada & USA fixed to 18,000ft
+        } else if (airportArrive_1 === "Y") {
+            SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", 10000); //SPA region only have 10,000ft & 11,000ft. But 10,000ft is more so using that
+        } else {
+            let arrival = airportData.find(airportData => airportData.icao === airportArrive_2);
+            if (arrival.icao !== airportArrive_2) {
+                SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", 18000); // if can't find on list, default value to 18,000ft
+            } else {
+                SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", arrival.transAlt);
+            }
+        }
+    }
+}
+
+//this list build manually so there can error in that list. you can fix that!
+const airportData = [
+    { icao: "AG", transAlt: 11000},
```

Comment:
```suggestion
    { icao: "AG", transAlt: 11000 },
```

Please fix formatting for every line